vault/resource_secrets_sync_association_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestSecretsSyncAssociation_gh(t *testing.T) {
+	mount := acctest.RandomWithPrefix("tf-test-sync")
+	destName := acctest.RandomWithPrefix("tf-sync-dest")
+	secretName := acctest.RandomWithPrefix("tf-sync-secret")
+
+	resourceName := "vault_secrets_sync_association.test"
+
+	values := testutil.SkipTestEnvUnset(t,
+		"GITHUB_ACCESS_TOKEN",
+		"GITHUB_REPO_OWNER",
+		"GITHUB_REPO_NAME",
+	)
+
+	accessToken := values[0]
+	repoOwner := values[1]
+	repoName := values[2]
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion115)
+		}, PreventPostDestroyRefresh: true,
+		Steps: []resource.TestStep{
+			{
+				Config: testSecretsSyncAssociationConfig_gh(mount, accessToken, repoOwner, repoName, destName, secretName),
+				Check: resource.ComposeTestCheckFunc(
+
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMount, mount),
```

Comment:
Nit: remove blank line
```suggestion
				Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr(resourceName, consts.FieldMount, mount),
```